documentation/api/io.strimzi.api.kafka.model.connect.build.Build.adoc

```diff
@@ -0,0 +1,191 @@
+Configures additional connectors for Kafka Connect deployments.
+
+=== `output`
+
+In order to build new container images with additional connector plugins, Strimzi requires a container registry where the images can be pushed to, stored and pulled from.
+Strimzi does not run its own container registry, so users need to provide their own registry.
+Strimzi is able to use private container registries as well as public registries such as link:https://quay.io/[Quay^] or link:https://hub.docker.com//[Docker Hub^].
+The container registry is configured in the `.spec.build.output` section of the `KafkaConnect` custom resource.
+The `output` configuration supports two types: `docker` and `imagestream`.
+The `output` section is required.
+
+.Using Docker registry
+
+To use a Docker registry, you have to specify the `type` as `docker`, and the `image` field with the full name of the new container image.
+The full name has to include:
+
+* The address of the registry.
```

Comment:
Does the address include organization and image name? Can we make it clearer?

Reply:
I'm not sure we should mentioned them specifically since they are concepts used by Docker Hub or Quay ... but many registries do not use them. So I think it might be confusing.